fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -810,18 +806,17 @@ public void disableVersioning() {
 
     }
 
+    public static Predicate<Node> isVersioned =
+            UncheckedPredicate.uncheck((final Node node) -> node.isNodeType("mix:versionable"));
```

Comment:
If you are going to do this, do we really need `(final Node node)`? Isn't just `node` enough for the compiler?